src/v/pandaproxy/client/fetch_session.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "kafka/types.h"
+#include "model/fundamental.h"
+
+#include <absl/container/flat_hash_map.h>
+
+namespace pandaproxy::client {
+
+class fetch_session {
```

Comment:
can we get a comment above the class on how this is integrated into the client library. 

what components use this, intention, etc.